src/solvers/dgsem_p4est/dg_2d_parabolic.jl

```diff
@@ -19,9 +19,7 @@ function create_cache_parabolic(mesh::P4estMesh{2}, equations_hyperbolic::Abstra
     return cache
 end
 
-# TODO: Remove in favor of the implementation for the TreeMesh 
-#       once the P4estMesh can handle mortars as well
```

Comment:
This is not easily possible as we call for `TreeMesh`es 

```julia
    # Prolong solution to mortars
    @trixi_timeit timer() "prolong2mortars" begin
        prolong2mortars!(cache, flux_viscous, mesh, equations_parabolic,
                         dg.mortar, dg.surface_integral, dg)
    end

    # Calculate mortar fluxes
    @trixi_timeit timer() "mortar flux" begin
        calc_mortar_flux!(cache_parabolic.elements.surface_flux_values, mesh,
                          equations_parabolic,
                          dg.mortar, dg.surface_integral, dg, cache)
    end
```

while for `P4estMesh`es we call


```julia
    # Prolong solution to mortars (specialized for AbstractEquationsParabolic)
    # !!! NOTE: we reuse the hyperbolic cache here since it contains "mortars" and "u_threaded"
    # !!! Is this OK?
    @trixi_timeit timer() "prolong2mortars" begin
        prolong2mortars_divergence!(cache, flux_viscous, mesh, equations_parabolic,
                                    dg.mortar, dg.surface_integral, dg)
    end

    # Calculate mortar fluxes (specialized for AbstractEquationsParabolic)
    @trixi_timeit timer() "mortar flux" begin
        calc_mortar_flux_divergence!(cache_parabolic.elements.surface_flux_values,
                                     mesh, equations_parabolic, dg.mortar,
                                     dg.surface_integral, dg, cache)
    end
```